src/Support/helpers.php

```diff
@@ -73,7 +73,7 @@ function get_sort(?string $property): string
      */
     function get_filter(string $property)
     {
-        $filter = new HttpFilter();
+        $filter = app(HttpFilter::class);
```

Comment:
why do we need DI inection for http filter?

Reply:
So it can be overriden when needed. eg. if an user wants to be able to expand filtersExact further (as I needed with arrays), they can do it. Without this extension, it needs a lots of hack. With adding DI they can just add a DI override within a service provider.

Reply:
not sure it's going to be approved like this. Please change it back as it's not a part of this mr